charts/agent-control/templates/_helpers.tpl

```diff
@@ -0,0 +1,7 @@
+{{- define "agent-control.release.name" -}}
+  {{- printf "%s-deployment" .Release.Name -}}
+{{- end -}}
+
+{{- define "agent-control.secret.name" -}}
+  {{- printf "%s-deployment" .Release.Name -}}
+{{- end -}}
```

Comment:
is it intended that these two functions do exactly the same? Should we unify them somehow?

Reply:
I did that on purpose. However, names could change if someone has a better idea. Thus, I decided to copy paste instead of reuse.

Reply:
If you want I could call the release one from the secret one. It wouldn't add nor remove any value, I think.

Reply:
no strong opinion regarding having two functions, but I still prefer one or reuse the code as you suggested 😅 your call

Reply:
I like having two. Just because they represent different things. The secret name and the release name. It's coincidental that the value is equivalent. Let me call one from the other though.